dashboards-observability/public/components/event_analytics/utils/utils.tsx

```diff
@@ -365,11 +365,9 @@ export const filterDataConfigParameter = (parameter: ConfigListEntry[]) =>
 export const getTooltipHoverInfo = ({ tooltipMode, tooltipText }: GetTooltipHoverInfoType) => {
   if (tooltipMode === 'hidden') {
     return 'none';
+  } else if (tooltipText === undefined) {
+    return 'all';
   } else {
-    if (tooltipText === undefined) {
-      return 'all';
-    } else {
```

Comment:
I think we don't need an else here just `return tooltipText;` as if it matches any branches before it would return already.

Reply:
it is updated in next commit